WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -436,6 +439,9 @@ public WordPressDB(Context ctx) {
             case 47:
                 db.execSQL(ADD_TESTIMONIALS_FLAG);
                 currentVersion++;
+            case 48:
```

Comment:
We should be able to tack `ADD_PORTFOLIOS_FLAG` onto version `47` changes since they haven't been merged into a release yet.